examples/ios/swift-2.2/GettingStarted.playground/Contents.swift

```diff
@@ -63,9 +67,15 @@ try! realm.write {
 
 let favorites = ["Jennifer"]
 
+#if swift(>=3.0)
+let favoritePeopleWithSpousesAndCars = realm.allObjects(ofType: Person.self)
+    .filter(using: "cars.@count > 1 && spouse != nil && name IN %@", favorites)
+    .sorted(onProperty: "age")
```

Comment:
Agreed. I think we ought to discuss the renaming again before Swift 3's final release.